docs/contracts/inheritance.rst

```diff
@@ -484,7 +484,31 @@ inheritance list or in modifier-style in the derived constructor.
 Specifying arguments in both places is an error.
 
 If a derived contract does not specify the arguments to all of its base
-contracts' constructors, it will be abstract.
+contracts' constructors, it must be declared abstract. In that case, when
+a contract derives from it, its declaration or its constructor must
```

Comment:
A declaration would be the whole contract. The text above already refers to this as "inheritance list" so we should stick to that term here:
```suggestion
a contract derives from it, its inheritance list or its constructor must
```